sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/continuous/EpochCoordinator.scala

```diff
@@ -117,7 +117,7 @@ private[sql] object EpochCoordinatorRef extends Logging {
 private[continuous] class EpochCoordinator(
     writeSupport: StreamingWrite,
     stream: ContinuousStream,
-    query: ContinuousExecution,
```

Comment:
I think the old naming was still right and this change just increasing the diff, as a `ContinuousExecution` is inherited from `StreamExecution` which is a `StreamingQuery`.

Or check for example `StreamingQueryWrapper` which has a member `_streamingQuery` with the type of `StreamExecution`.

Reply:
reverted